main.go

```diff
@@ -58,8 +58,14 @@ func main() {
 
 	// get plan
 	ctx := context.Background()
+	apiClient := api.NewClient(api.ClientConfig{
+		ServerBaseUrl:    cfg.ServerBaseUrl,
+		AccessToken:      cfg.AccessToken,
+		OrganizationSlug: cfg.OrganizationSlug,
+		Version:          cfg.Version,
```

Comment:
I saw you removed `version` from config in your other PR https://github.com/buildkite/test-splitter/pull/99/files. Do we need to apply the same change here?

Reply:
Good point!